crates/compiler/can/src/builtins.rs

```diff
@@ -191,8 +191,8 @@ map_symbol_to_lowlevel_and_arity! {
     NumShiftRightZfBy; NUM_SHIFT_RIGHT_ZERO_FILL; 2,
     NumToStr; NUM_TO_STR; 1,
 
-    Eq; BOOL_EQ; 2,
-    NotEq; BOOL_NEQ; 2,
+    Eq; EQ_STRUCTURAL_EQ; 2,
+    NotEq; EQ_STRUCTURAL_NOT_EQ; 2,
```

Comment:
Again names suggest there's an Eq module but there doesn't seem to be.

Reply:
This was intentional, but I can see how it's confusing. Will change